mad-dog/README.md

```diff
@@ -14,14 +14,25 @@
 **Note:** If tests fail, sometimes running `npm i` in `/libs` and relinking packages will help. Or, bring up a fresh set of local services.
 
 ### Running Tests
-`npm run start test` or `npm run start test-ci` (depending on the test suite you want to run)
+<!-- `npm run start test` or `npm run start test-ci` (depending on the test suite you want to run)
+Add the cli argument `verbose` to print out the docker logs of the time of any failed tests.
+i.e. `npm run start test <offset> verbose` -->
+**Run all the tests in test suite**: 
+- `npm run start <test name>`
+
+**Run all the tests in test suite with specified account offset**: 
+- `npm run start <test name> <0-99>`
+
+**Run all the tests in test suite in verbose mode (prints out container logs if tests fail)**: 
+- `npm run start <test name> verbose` 
+- `npm run start <test name> <0-99> verbose` 
```

Comment:
what's the <0-99> mean here? (i know what it means, i think we should be a bit clearer if we're already touching the README)

Reply:
it's the account offset in the wallet. i'll update the README 👍